sketch/payload_function.js

```diff
@@ -0,0 +1,35 @@
+function Decoder(bytes, port) {
+
+  var decoded = {};
+
+  bssid = shiftBytes(bytes);
+  decoded.Bssid1 = bssid.join('');
+
+  bssid = shiftBytes(bytes);
+  decoded.Bssid2 = bssid.join('');
+
+  bssid = shiftBytes(bytes);
+  decoded.Bssid3 = bssid.join('');
+
+  return decoded;
+}
+
+function pad(n) {
+  return (n < 10 || n == 'a' || n == 'b' || n == 'c' || n == 'd' || n == 'e' || n == 'f') ? ("0" + n) : n;
+}
+
+function shiftBytes(byteArray) {
+  var i, bit;
+  var tempArray = [6];
```

Comment:
This initializes an array with one entry `6`.

I guess you meant an array of size 6?

Reply:
yes, that was was the point. 
The payload function you wrote was the aim i had, but i couldn't manage to get it the way you have made it. Look good! I will add it in.